website/docs/sanitize.md

```diff
@@ -12,6 +12,8 @@ sanitize { "value_list" }
 |-------------|--------------------------------------------------------|
 | Address     | Enables compiler support for AddressSanitizer. | Visual Studio support starts with 2019 16.9 |
 | Fuzzer      | Enables support for LibFuzzer, a coverage-guided fuzzing library. | Visual Studio support starts with 2019 16.9 |
+| Thread      | Enables compiler support for ThreadSanitizer. |
+| Undefined   | Enables compiler support for UndefinedBehaviorSanitizer (UBSan). |
```

Comment:
supported only by gcc/clang

Check also if msvc with clang toolset can support it

Reply:
`clang-cl` supports passing `-fsanitize=undefined` to it, but I am unable to find a corresponding `vcxproj` flag for it.

Reply:
Just keeping it with gcc/clang without MSVC/MSBuild support is perfectly sufficient for now.